libc/newhdrgen/yaml_to_classes.py

```diff
@@ -23,6 +22,11 @@
 from class_implementation.classes.object import Object
 
 
+class MyDumper(yaml.Dumper):
```

Comment:
1. Consider renaming this to something more descriptive (maybe `IndentListDumper`?)
2. If you don't need this class anywhere else, you can define it in `add_function_to_yaml(...)` to reduce its scope.